src/main/java/tahub/contacts/MainApp.java

```diff
@@ -90,7 +93,7 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             initialData = new AddressBook();
         }
 
-        return new ModelManager(initialData, userPrefs);
+        return new ModelManager(initialData, userPrefs, new tahub.contacts.model.course.UniqueCourseList());
```

Comment:
Why not just import it and call `UniqueCourseList()`?

Reply:
fixed